db/seeds.d/02-roles_list.rb

```diff
@@ -79,16 +50,7 @@ def manage_organizations_permissions
     end
 
     def view_permissions
-      [
-        :view_architectures, :view_authenticators, :view_bookmarks, :view_compute_resources, :view_compute_resources_vms, :view_compute_profiles,
-        :view_config_groups, :view_domains, :view_realms, :view_environments, :view_external_usergroups, :view_external_variables,
-        :view_external_parameters, :view_filters, :view_hostgroups, :view_keypairs,
-        :view_hosts, :view_params, :view_images, :view_locations, :view_media, :view_models, :view_operatingsystems,
-        :view_provisioning_templates, :view_ptables, :view_puppetclasses, :view_smart_proxies, :view_smart_proxies_autosign,
-        :view_smart_proxies_puppetca, :view_subnets, :view_organizations, :view_usergroups, :view_users, :view_config_reports,
-        :view_facts, :view_audit_logs, :view_statistics, :view_tasks, :view_trends, :view_plugins, :view_mail_notifications,
-        :access_dashboard, :view_roles, :view_ssh_keys
-      ]
+      PermissionsList.permissions.select { |resource, name| name.start_with?('view_') }.map(&:last)
```

Comment:
this wasn't a problem, there's `map(&:last)` it worked with adding `seeded_roles` but for consistency I think it will be better to use the same construct as in `base_manage_permissions`